source/compiler/tests/error_094.pwn

```diff
@@ -0,0 +1,20 @@
+#include <console>
+
+main()
+{
+	// Case 1: Both operands are compile-time constants
+	printf("%d", 1 / 0); // error 094
+	printf("%d", 1 % 0); // error 094
+	printf("%d", 1 / 1);
+	printf("%d", 1 % 1);
+
+	// Case 2: Only the divisor is a constant
+	new var = 1;
+	printf("%d", var / 0); // error 094
+	printf("%d", var % 0); // error 094
+	printf("%d", var / 1);
+	printf("%d", var % 1);
+
+	printf("%d", 1 / var); // Just to make sure the warning works only
```

Comment:
I sometimes hate the github review system.  I made that comment literally MONTHS ago, but because I forgot to click "submit review", it just sat there unpublished until I noticed just now...

Reply:
Ah, right. Thanks for pointing it out!